README.md

```diff
@@ -2,115 +2,91 @@
 
 VPF stands for Video Processing Framework. It’s set of C++ libraries and Python bindings which provides full HW acceleration for video processing tasks such as decoding, encoding, transcoding and GPU-accelerated color space and pixel format conversions.
 
-VPF also supports exporting GPU memory objects such as decoded video frames to PyTorch tensors without Host to Device copies. Check the [Wiki page](https://github.com/NVIDIA/VideoProcessingFramework/wiki/Building-from-source) on how to build from source.
-
-## Installation
-
-VPF works on Windows and Linux. The requirements are as follows
-
-- CUDA Toolkit (npp)
-- [FFMPEG](https://github.com/FFmpeg/FFmpeg/) (with libavfilter>=7.110.100)
-- C++ compiler
-
-### Linux
-
-We recommend Ubuntu 22.04 as it comes with a recent enough FFmpeg system packages.
-If you want to build FFmpeg from source, you can follow
-https://docs.nvidia.com/video-technologies/video-codec-sdk/12.0/ffmpeg-with-nvidia-gpu/index.html
-```bash
-# Install dependencies
-apt install -y \
-          libavfilter-dev \
-          libavformat-dev \
-          libavcodec-dev \
-          libswresample-dev \
-          libavutil-dev\
-          wget \
-          build-essential \
-          git
-
-# Install CUDA Toolkit (if not already present)
-wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.0-1_all.deb
-sudo dpkg -i cuda-keyring_1.0-1_all.deb
-sudo apt-get update
-sudo apt-get install -y cuda
-# Ensure nvcc to your $PATH (most commonly already done by the CUDA installation)
-export PATH=/usr/local/cuda/bin:$PATH
-
-# Install VPF
-pip3 install git+https://github.com/NVIDIA/VideoProcessingFramework
-# or if you cloned this repository
-pip3 install .
-```
-
-To check whether VPF is correctly installed run the following Python script
-```python
-import PyNvCodec
-```
-If using Docker via [Nvidia Container Runtime](https://developer.nvidia.com/nvidia-container-runtime),
-please make sure to enable the `video` driver capability: https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/user-guide.html#driver-capabilities via
-the `NVIDIA_DRIVER_CAPABILITIES` environment variable in the container or the `--gpus` command line parameter (e.g.
+## Prerequisites
+
+- NVIDIA display driver: R525 or above
+- CUDA Toolkit 11.8 or above 
+  - CUDA toolkit has driver bundled with it e.g. CUDA Toolkit 12.0 has driver r530. During installation of CUDA toolkit you could choose to install or skip installation of the bundled driver. Please choose the appropriate option.
+- FFMPEG (with libavfilter >=7.110.100)
+- Compile FFMPEG with shared libraries or download pre-compiled binaries from a source you trust. During VPF’s “pip install”(mentioned in sections below) we need to provide a path to the directory where FFMPEG got installed.
+- CMake 3.14 or above
```

Comment:
We actually require CMake 3.21. Since we automatically install cmake with the right version now with pip now for the user I would just omit this requirement.

Reply:
done